src/connectionhealthpolicy/ConnectionHealthData.ts

```diff
@@ -13,6 +13,11 @@ export default class ConnectionHealthData {
   packetsReceivedInLastMinute: number[] = [];
   fractionPacketsLostInboundInLastMinute: number[] = [];
   audioSpeakerDelayMs = 0;
+  isVideoEncoderHardware = false;
+  videoEncodingTimeInMs = 0;
+  cpuLimitationDuration = 0;
+  videoInputFps = 0;
+  videoEncodeFps = 0;
```

Comment:
if we ever do a major version...we should really just pass the whole metrics report...

Reply:
Agree. We may need to do a cleanup here later